modules/cnf-topology-aware-lifecycle-manager-preparing-for-updates.adoc

```diff
@@ -40,12 +40,14 @@ imageContentSources:
 $ OCP_RELEASE_NUMBER=<release_version>
 ----
 
-.. Specify the architecture of the server by running the following command:
+.. Specify the architecture of the cluster by running the following command:
 +
 [source,terminal]
 ----
-$ ARCHITECTURE=<server_architecture>
+$ ARCHITECTURE=<cluster_architecture> <1>
 ----
+<1> For `cluster_architecture`, specify the architecture of the cluster, such as `x86_64`, `aarch64`, `s390x`, `ppc64le`, or `multi`.
```

Comment:
```suggestion
<1> Specify the architecture of the cluster, such as `x86_64`, `aarch64`, `s390x`, `ppc64le`, or `multi`.
```
Minimalism suggestion for consistency with other code blocks (such as annotations 6-9 here: https://docs.openshift.com/container-platform/4.13/installing/installing_azure/installing-azure-customizations.html#installation-azure-config-yaml_installing-azure-customizations) 

You can apply the same change to the rest of the modules rather than me leaving 6 more comments :)